tests/collections/asr/decoding/test_fast_rnnt_greedy_decoding.py

```diff
@@ -0,0 +1,187 @@
+import glob
+import json
+import os
+import sys
+import tempfile
+import traceback
+
+import ipdb
+import jiwer
+import pytest
+import torch
+from omegaconf import OmegaConf, open_dict
+
+from nemo.collections.asr.models import ASRModel
+from nemo.collections.asr.models.rnnt_bpe_models import EncDecRNNTBPEModel
+from nemo.collections.asr.parts.submodules.fast_rnnt_greedy_decoding import RNNTGreedyDecodeFast
+from nemo.collections.asr.parts.submodules.rnnt_greedy_decoding import GreedyBatchedRNNTInfer
+
+
+@pytest.mark.parametrize(
+    ("model_name", "batch_size", "use_subset", "enable_bfloat16"),
+    [
+        # The xfail is not catching for some reason...
+        # pytest.param("stt_en_fastconformer_transducer_large", 8, True, True, marks=pytest.mark.xfail(reason="Cannot instantiate graph with persistent RNN")),
```

Comment:
Is `xlarge` model different from `large`? Maybe add `TODO` for this?

Reply:
Yes, the LSTM is different in terms of its parameter size. "large" triggers a persistent RNN kernel in cudnn, which is a cooperative kernel launch. And cooperative kernel launches are disallowed inside of the body graphs of conditional nodes in cuda graphs right now.